docs/ops/deployment/yarn.md

```diff
@@ -0,0 +1,203 @@
+
+### Configuring Flink on YARN
+
+The YARN-specific configurations are listed on the [configuration page](http://localhost:4000/ops/config.html#yarn).
+
+The following configuration parameters have no effect in Flink on YARN, as they are overwritten by the framework at runtime:
+- `jobmanager.rpc.address`
+- `io.tmp.dirs` (Flink sets the temporary directories defined by YARN)
+
+
+#### Configuration handling of `bin/yarn-session.sh`
+
+Besides passing configuration via the `conf/flink-conf.yaml` file, you can also pass any configuration at submission time to the `./bin/yarn-session.sh` client using `-Dkey=value` arguments.
+
+The YARN session client also has a few "shortcut arguments" for commonly used settings. They can be listed with `./bin/yarn-session.sh -h`.
+
+#### Configuration handling of `bin/flink run -m yarn-cluster`
+
+With `bin/flink run -m yarn-cluster`, you can run the command line interface in YARN cluster mode, allowing to submit jobs to an ad-hoc YARN cluster. The command line options of the YARN session described above are then also available. They are prefixed with a `y` or `yarn` (for the long argument options).
+
+
+### Resource Allocation Behavior
+
+A JobManager submitted in YARN session mode will wait for job submissions before starting to request and allocate TaskManagers. The other deployment modes are immediately requesting the required TaskManagers from YARN.
+
+The memory configurations for JobManager and TaskManager processes will be respected by the YARN implementation. The number of reported VCores is by default equal to the number of configured slots per TaskManager. The [`yarn.containers.vcores`]({% link ops/config.md %}#yarn-containers-vcores) allows overwriting the number of vcores with a custom value. In order for this parameter to work you should enable CPU scheduling in your cluster.
+
+Failed containers (including the JobManager) are replaced by YARN. The maximum number of JobManager container restarts is configured via `yarn.application-attempts` (default 1). The YARN Application will fail once all attempts are exhausted.
+
+
+### High-Availability on YARN
+
+For enabling HA on YARN, the `yarn.application-attempts` need to be greater 1, and a [high availability service needs to be configured]().
+
+### Supported Hadoop versions.
+
+Flink on YARN is compiled against Hadoop 2.4.1, which is also the earliest version we are supporting. 
+
+For providing Flink with the required Hadoop dependencies, we recommend setting the `HADOOP_CLASSPATH` environment variable. 
+If that is not possible, the dependencies can also be put into the `lib/` folder of Flink. Flink also offers pre-bundled Hadoop fat jars for placing them in the `lib/` folder, in the "Downloads / Additional Components" section of the website. These pre-bundled fat jars are shaded to avoid dependency conflicts with common libraries.
+
+The Flink community is not testing the YARN integration against these pre-bundled jars. 
```

Comment:
```suggestion
If that is not possible, the dependencies can also be put into the `lib/` folder of Flink. 

Flink also offers pre-bundled Hadoop fat jars for placing them in the `lib/` folder, in the "Downloads / Additional Components" section of the website. These pre-bundled fat jars are shaded to avoid dependency conflicts with common libraries. The Flink community is not testing the YARN integration against these pre-bundled jars. 
```

I would organize the paragraphs differently: The first paragraph is about integrating Hadoop. The second paragraph is about using pre-bundled jars.

Additionally, is there a reason why we don't link the Downloads page?